README.md

```diff
@@ -25,3 +25,18 @@ Pull request name contains issue number in the end.
 Commit description structure:
 
 `<Verb infinitive> {grammatical modifiers for subject} <subject> #<ZenHub issue number>`
+
+## Development tools
++ <b>Python</b> 3.8 and later for bot and website backend
```

Comment:
It is a better to have a links to official resources. See [this](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#:~:text=Or%20pluses-,Links,-There%20are%20two)